pages/_formats/genome-coordinates.md

```diff
@@ -104,14 +102,22 @@ Using fully-inclusive human-readable notation leads to inferior representations
 Consider some feature lying on an exon, whose location is represented as \[fs,fe) relative to the start of the exon.
 In turn, the exon is located at coordinates \[es,ee) on a reference chromosome:
 
-<img src="{{ 'assets' | relative_url }}/img/genome-coordinates-img/nested.svg" alt="Feature nested on an exon" />
+![Feature nested on an exon](/assets/img/genome-coordinates-img/nested.svg)
 
 With 0-based coordinates, calculating the feature's location on the chromosome is simple: \[fs+es,fe+es).
 
 Similarly to finding the length of a subsequence in an inclusive notation, in 1‑based coordinates this calculation would require carefully subtracting 1 as appropriate.
 When strandedness comes into play and the feature's exon-coordinates are perhaps reversed, tracking the appropriate places to add and subtract 1 becomes harder.
 Thus 1-based coordinates are more susceptible to _off-by-one_ programming errors.
 
+### Existing GA4GH products
```

Comment:
I am +1 for @andrewyatz table here. The page is __not__ primarily for "this is how it should be", but, well, 50/50 "this is how it is / should be (with arguments)".

Reply:
The text does describe how it is. As noted in https://github.com/ga4gh-schemablocks/ga4gh-schemablocks.github.io/pull/4#issuecomment-465218193, the table is inaccurate for SAM/BAM/CRAM/VCF/BCF and refget and Beacon.

But more to the point: for a protocol, the table's separate Interbase vs. 0-start half-open columns are a distinction without a difference — the protocol has start/end fields, and clients and implementers are free to think of them as 0-start half-open or interbase as they wish.

So I am -1 for the table because once these issues are corrected, I don't think it adds much information.